client/lib/post-normalizer/test/index.js

```diff
@@ -1,41 +1,11 @@
-require( 'lib/react-test-env-setup' )();
-
 /**
  * External dependencies
  */
-
-// Uncomment this require to use the blanket coverage tests.
-//require( 'blanket' )( {
-//  pattern: /post-normalizer\//
-//} );
-
 const assert = require( 'chai' ).assert,
 	Spy = require( 'sinon' ).spy;
-/**
- * Internal dependencies
- */
-let normalizer = require( '../' ),
-	safeImageUrlFake = require( 'lib/safe-image-url' ),
-	allTransforms = [
-		normalizer.decodeEntities,
-		normalizer.stripHTML,
-		normalizer.preventWidows,
-		normalizer.makeSiteIDSafeForAPI,
-		normalizer.pickPrimaryTag,
-		normalizer.safeImageProperties( 200 ),
-		normalizer.withContentDOM(),
-		normalizer.withContentDOM( [
-			normalizer.content.removeStyles,
-			normalizer.content.safeContentImages( 300 ),
-			normalizer.content.makeEmbedsSecure,
-			normalizer.content.detectEmbeds,
-			normalizer.content.wordCountAndReadingTime
-		] ),
-		normalizer.createBetterExcerpt,
-		normalizer.waitForImagesToLoad,
-		normalizer.pickCanonicalImage,
-		normalizer.keepValidImages( 1, 1 )
-	];
+
+require( 'lib/react-test-env-setup' ).auto();
```

Comment:
I did some testing around that solution. It works properly at the moment, because we have only test file in this folder. With the setup we have at the moment, it would be enabled during execution of all tests required in this folder. Personally I would move it inside `describe` function and put it explicitly for all tests that really need this kind of setup.